modules/fg_util.py

```diff
@@ -71,31 +72,30 @@ def AllFactions ():
 def ReadBaseNameList(faction):
     bnl=[]
     debug.debug('reading base names %s', faction)
-    filename = 'universe/fgnames/'+faction+'.txt'
-    try:
-        f = open (filename,'r')
-        bnl = f.readlines()
-        f.close()
-    except:
+    filename = os.path.join('universe', 'fgnames', faction + '.txt')
+    base_faction = faction.replace('_citizen', '').replace('_guild', '').replace('_briin', '').replace('n_merchant_marine', '')
+    base_filename = os.path.join('universe', 'fgnames', base_faction + '.txt')
+    names = [
+             filename,
+             base_filename,
+             os.path.join('../', filename),
+             os.path.join('../', base_filename),
+             os.path.join('universe', 'fgnames', 'names.txt'),
+             os.path.join('..', 'universe', 'names.txt'),
+             os.path.join('universe', 'names.txt')
+            ]
+    for name in names:
         try:
-            f = open ('../'+filename,'r')
-            bnl = f.readlines()
-            f.close()
-        except:
-            try:
-                f = open ('../universe/names.txt','r')
+            with open (name,'r') as f:
```

Comment:
Looks like on Windows, you have to specify a code page, or else Python 3 will default to cp1252 for some reason.

Try replacing this line with:

```py
with open (name, 'r', encoding='utf8') as f:
```

Reply:
@stephengtuggy Done, of course it keeps working on Linux as well.